platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/reconnect/RandomVirtualMapMerkleDbReconnectTests.java

```diff
@@ -67,38 +64,15 @@ class RandomVirtualMapMerkleDbReconnectTests extends VirtualMapReconnectTestBase
     public static final String LETTERS = "abcdefghijklmnopqrstuvwxyz";
     public static final int ZZZZZ = 26 * 26 * 26 * 26 * 26; // key value corresponding to five Z's (plus 1)
 
-    private static MerkleDbSettings originalSettings;
-
     @BeforeAll
-    static void beforeAll() throws Exception {
-        originalSettings = MerkleDbSettingsFactory.get();
-        MerkleDbSettingsFactory.configure(new TestMerkleDbSettings(originalSettings) {
-            @Override
-            public int getKeySetBloomFilterHashCount() {
-                return 10;
-            }
-
-            @Override
-            public long getKeySetBloomFilterSizeInBytes() {
-                return 2 * MEBIBYTES_TO_BYTES * BYTES_TO_BITS;
-            }
-
-            @Override
-            public long getKeySetHalfDiskHashMapSize() {
-                return 10_000;
-            }
-
-            @Override
-            public int getKeySetHalfDiskHashMapBuffer() {
-                return 1_000;
-            }
-        });
-        ConstructableRegistry.getInstance().registerConstructables("com.swirlds.platform");
-    }
-
-    @AfterAll
-    static void afterAll() {
-        MerkleDbSettingsFactory.configure(originalSettings);
+    static void beforeAll() {
+        final Configuration config = new TestConfigBuilder()
+                .withValue("merkleDb.keySetBloomFilterSizeInBytes", "16777216")
```

Comment:
Is the `"16777216"` value meaningful in some way and can be written as a factor of parts that illustrate the meaning?

Reply:
Given that @ConfigProperty doesn't support the use of expressions or constants such as `2 * MEBIBYTES_TO_BYTES * BYTES_TO_BITS`, I found it necessary to manually compute the value, resulting in 2 * 1024 * 1024 * 8 = 16777216.

Fortunately, the withValue function doesn't have this constraint, thus enabling me to revert to using the previous constants.